mindjunkies/courses/models.py

```
@@ -1,11 +1,16 @@
+import cloudinary
+import cloudinary.uploader
```

Ruff (0.8.2)

`cloudinary.uploader` imported but unused

Remove unused import: `cloudinary.uploader`

(F401)